src/main/scripts/org/reaktivity/specification/nukleus/http/streams/rfc7230/architecture/request.and.response/raw/server.rpt

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+# source (cooked) -> http -> target (this script) -> network
+# Read from target/http, write to http/target
+
+property source "http"
+property newTargetOutputRef ${nuklei:newReferenceId()} # external scope
+       
+accept http://localhost:8080/ option http:transport nukleus://streams/target
+       option nukleus:source ${source}
+       option nukleus:route ${newTargetOutputRef}
+       option nukleus:window 8192
+       option nukleus:transmission "duplex"
```

Comment:
I am not sure if the nukleus: options will automatically be applied as options to the http:transport.

Ideally it would be possible to parameterize the transport URI and options, something like:
```
accept http://localhost:8080/ option ${transportURI} ${transportOptions}
```
so the script can be used against any HTTP client